src/main/scala/com/rawlabs/das/server/DASSdkManager.scala

```diff
@@ -41,6 +41,7 @@ class DASSdkManager(implicit settings: RawSettings) extends StrictLogging {
   private val dasSdkLoader = ServiceLoader.load(classOf[DASSdkBuilder]).asScala
 
   private val dasSdksInMemory = mutable.HashMap[DASId, DaSDKInMemoryEntry]()
+  private val dasSdksPerConfig = mutable.HashMap[Map[String, String], DASSdk]()
   private val dasSdksInMemoryLock = new Object
```

Comment:
Suggest you rename to dasSdksLock since now it's locking more than just dasSdksInMemory